rest/blip_sync.go

```diff
@@ -390,6 +390,14 @@ func (bh *blipHandler) sendRevision(seq db.SequenceID, docID string, revID strin
 	outrq := blip.NewRequest()
 	outrq.SetProfile("rev")
 	seqJSON, _ := json.Marshal(seq)
+	outrq.Properties["id"] = docID
+	delete(body, "_id")
+	outrq.Properties["rev"] = revID
+	delete(body, "_rev")
+	if del, _ := body["_deleted"].(bool); del {
+		outrq.Properties["deleted"] = "1"
+		delete(body, "deleted")
+	}
```

Comment:
Yeah, that would be good. Of course that GetRev would need to return an extra boolean parameter for the `deleted` flag.